cmd/ppcheck.js

```diff
@@ -16,25 +16,44 @@ module.exports.run = (client, message, args, maindb) => {
 	binddb.find(query).toArray(function(err, res) {
 		if (err) throw err;
 		if (res[0]) {
+			var uid = res[0].uid;
 			var username = res[0].username;
 			var discordid = res[0].discordid;
 			var pp = 0;
 			var ppentry = [];
 			if (res[0].pptotal) pp = res[0].pptotal.toFixed(2);
 			if (res[0].pp) ppentry = res[0].pp;
-			
-			var output = '<@' + discordid + '> PP Profile for ' + username + '\n';
-			var codeoutput = 'Total PP: ' + pp + '\n'; 
+
+			let site = "[PP Profile](https://ppboard.herokuapp.com/profile?uid=" + uid + ")";
+			let mirror = "[Mirror](https://droidppboard.herokuapp.com/profile?uid=" + uid + ")";
+
+			const embed = new Discord.RichEmbed()
+				.setDescription('**PP Profile for <@' + discordid + '> (' + username + ')**\nTotal PP: **' + pp + " pp**\n" + site + " - " + mirror)
+				.setColor(message.member.highestRole.hexColor)
+				.setFooter("Elaina owo", "https://image.frl/p/yaa1nf94dho5f962.jpg");
+
 			for (var x = 0; x < 5; x++) {
-				if (ppentry[x]) codeoutput += (x+1) + '. ' + ppentry[x][1] + ' - ' + ppentry[x][2] + ' pp\n';
-				else codeoutput += '-\n';
+				if (ppentry[x]) {
+					let combo = ppentry[x][3].toString();
+					if (combo.indexOf("x") == -1) combo = combo + "x";
+
+					let acc = ppentry[x][4].toString();
+					if (acc.indexOf('\r') != -1) acc = acc.replace(" ", "").replace("\r", "");

```

Comment:
This part is necessary to reformat accuracy string because acc submission can be different for each play